common/create-tower-catalog-item.adoc

```diff
@@ -13,20 +13,19 @@ First, create a catalog:
 
 Then, create an Ansible Tower service catalog item:
 
-. Navigate to menu:Automation[Ansible Tower > Jobs].
-. Click *Ansible Tower Job Templates* and select an Ansible Tower job template.
-. Click  image:1847.png[Configuration] (*Configuration*), then  image:1862.png[Create Service Dialog from this Job Template] (*Create Service Dialog from this Job Template*).
+. Navigate to menu:Automation[Ansible Tower > Explorer], then click on the *Templates* according menu.
```

Comment:
Should "according" be "accordion"? Also, when doing a quick runthrough on our test instance, this was called *Job Templates* -- perhaps this is an older instance though at .199?